src/classes/Contacts.cls

```diff
@@ -174,6 +245,60 @@ public inherited sharing class Contacts extends fflib_SObjects {
         }
     }
 
+    public void onAfterDelete_ACCT_IndividualAccounts_TDTM(TDTM_Runnable.DmlWrapper dmlWrapper) {
+        HouseholdAccounts householdAccounts = getHouseholdsFor(accountIds());
+        householdAccounts.handleContactDeletion(dmlWrapper);
+    }
+
+    public void onAfterUndelete(
+            TDTM_Runnable.DmlWrapper dmlWrapper) {
+        /**
+         * @description In the after trigger context, additional fields from the account
+         * related to the contact are queried for processing
+         * household updates.
+         */
+        List<Contact> contactsWithAccountAndAddressFields =
+                getContactsWithAccountAndAddressFields();
+
+        Households.updatePrimaryContactOnAccountsAfterInsert(
+                dmlWrapper, contactsWithAccountAndAddressFields);
+
+        HouseholdAccounts.renameHouseholdAccountsAfterInsert(
+                contactsWithAccountAndAddressFields);
+    }
+
+    /**
+     * @description Determines whether or not Contacts belong to the BucketAccount, when
+     * Contacts are being inserted.
+     */
+    public Contacts bucketContacts_insertContext() {
+        return new Contacts(contactsWithoutAccounts());
+    }
+
+    public Contacts contactsAndOneToOneContacts_insertContext() {
```

Comment:
In our variable and method names that are related to the 1:1 model, we use the naming convention of either `OneToOne` or `One2One`. For searchability (and perhaps other benefits), do you think it makes sense to align on a singular naming convention here?

Reply:
Yep.  Should prob standardize on OneToOne, oneToOne, One-to-one, etc.

<img width="1041" alt="2021-07-13_16-53-51" src="https://user-images.githubusercontent.com/46458081/125539860-025753cb-b0d2-40b4-8642-ace99994118b.png">